middleware/logger/logger.go

```diff
@@ -137,56 +134,6 @@ func New(config ...Config) fiber.Handler {
 		// Get new buffer
 		buf := bytebufferpool.Get()
 
-		// Default output when no custom Format or io.Writer is given
-		if cfg.Format == ConfigDefault.Format {
```

Comment:
not sure, is this part no longer needed ? where is this code now covered ?

Reply:
thanks, this old colorization is therefore superfluous, since the enabledColors are queried in the methods for the tags